### PR TITLE
Translate models.md's en version into chinese

### PR DESCRIPTION
**Pull request by wslgqq277g**
translation work has been done, please check it.

### CODE REVIEW

File: docs/zh_cn/migration/models.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+</td>
+
+</tr>
+</thead>
+</table>
+
+我们在 MMagic 1.x. 版本中对模型进行了重构，其中重要的改动如下所示：
+
+MMagic 1.x 中的 models 被重构为六个部分：archs、base_models、data_preprocessors、editors、diffusion_schedulers 和 losses。
```
Comment (zengyh1900) [inline docs/zh_cn/migration/models.md:68]:
```suggestion
- MMagic 1.x 中的 `models` 被重构为六个部分：`archs`、`base_models`、`data_preprocessors`、editors、`diffusion_schedulers` 和 `losses`。
```

File: docs/zh_cn/migration/models.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+在 models 中添加了 data_preprocessor 模块。归一化和颜色空间转换操作从 dataset transform 流程中移动到 data_preprocessor 中。数据流从数据管道中输出后，先经过 data_preprocessor 模块的转换，然后再输入模型中。
```
Comment (zengyh1900) [inline docs/zh_cn/migration/models.md:69]:
```suggestion
- 在 `models` 中添加了 `data_preprocessor` 模块。这里主要是将归一化和颜色空间转换操作从 `dataset transform` 流程中移动到 `data_preprocessor` 中。此时，数据流经过数据预处理后，会先经过 `data_preprocessor` 模块的转换，然后再输入到模型中。
```

**Review by zengyh1900 — CHANGES_REQUESTED** [review]
Thanks for your contribution! It looks good to me!
I have suggested several changes to make it better. After the change, we can merge this pull request.

File: docs/zh_cn/migration/models.md [hunk 3]
```diff
@@ -1 +1,70 @@
-# 模型的迁移（待更新）
+# 模型的迁移
+
+我们在 MMagic 1.x. 版本更新了模型设定，其中重要的改动如下所示：
+
+- 删除 `pretrained` 字段.
+- 在模型设定中添加 `train_cfg` 和 `test_cfg` 字段.
+- 添加 `data_preprocessor` 字段. 归一化和颜色空间变换操作从datasets transforms流程中转到data_preprocessor.我们接下来会介绍data_preprocessor。
```
Comment (wslgqq277g) [inline docs/zh_cn/migration/models.md:7]: - 添加 `data_preprocessor` 字段. 归一化和颜色空间变换操作从datasets transforms流程中转到data_preprocessor.我们接下来会介绍data_preprocessor。
+-- 添加 `data_preprocessor` 字段. 这里主要是将归一化和颜色空间转换操作从 `dataset transform` 流程中移动到 `data_preprocessor` 中.我们接下来会介绍`data_preprocessor`.

Comment (zengyh1900) [reply docs/zh_cn/migration/models.md]: hi，现在文档部分似乎没有修改过来，你需要再修改一下。

Comment (zengyh1900) [reply docs/zh_cn/migration/models.md]: 另外， 可以参考 https://mmagic.readthedocs.io/zh_CN/latest/community/contributing.html#id13 ，通过 pre-commit 来修正 lint（代码风格）的问题

**Review by zengyh1900 — APPROVED** [review]
lgtm.

**Comment by OpenMMLab-Assistant-004** [comment]
Hi @wslgqq277g,

We'd like to express our appreciation for your valuable contributions to the mmagic. Your efforts have significantly aided in enhancing the project's quality.
It is our pleasure to invite you to join our community thorugh Discord_Special Interest Group (SIG) channel. This is a great place to share your experiences, discuss ideas, and connect with other like-minded people. To become a part of the SIG channel, send a message to the moderator, OpenMMLab, briefly introduce yourself and mention your open-source contributions in the #introductions channel. Our team will gladly facilitate your entry. We eagerly await your presence. Please follow this link to join us: ​https://discord.gg/UjgXkPWNqA.

If you're on WeChat, we'd also love for you to join our community there. Just add our assistant using the WeChat ID: openmmlabwx. When sending the friend request, remember to include the remark "mmsig + Github ID".

Thanks again for your awesome contribution, and we're excited to have you as part of our community!

**Pull request merged**